### ISSUE TITLE
[Bug] text doubling

### ISSUE THREAD

**Issue by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2070]
### Checklist

- [X] I am using the **latest v2 version**
- [X] I have searched [FAQ](https://vuepress-theme-hope.github.io/v2/faq.html)

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [ ] macOS
- [X] Windows
- [ ] Linux

### Describe the bug

After several page reloads the text in the sidebar and in the title is doubled:

![image](https://user-images.githubusercontent.com/63556979/179193863-1e1ebaec-629d-403d-9a03-dabfee40238e.png)

**Comment by tadmi** [comment]
**sidebar**

```
export default sidebar([
	{
		prefix: 'os',
		path: 'os/',
		text: 'ТРЕБОВАНИЯ',
		link: 'os/',
		icon: 'file-shield',
		children: [
			'common',
			'pc',
			'sanitation',
			'fire',
			'electric',
			'move',
			{
				prefix: 'prod/',
				path: 'prod/',
				text: 'Производство работ',
				link: 'prod/',
				icon: 'person-digging',
			},
		],
	},
	{
		prefix: 'actions',
		link: 'actions/',
		path: 'actions/',
		icon: 'angles-right',
		text: 'АЛГОРИТМЫ ДЕЙСТВИЙ',
		children: ['incident', 'first-aid'],
	},
	{
		path: 'signs/',
		link: 'signs/',
		text: 'ЗНАКИ',
		icon: 'ban-smoking',
	},
]);
```

**hopeTheme**

```
export default hopeTheme({
	iconAssets: 'fontawesome',
	navbar,
	home: '/os/',
	sidebar,
	darkmode: 'toggle',
	themeColor: {
		blue: '#2196f3',
		red: '#f26d6d',
		green: '#3eaf7c',
		orange: '#fb9b5f',
	},
	fullscreen: true,
	lastUpdated: false,
	editLink: false,
	contributors: false,
	pageInfo: [],
	plugins: {
		components: ['Badge', 'YouTube'],
		photoSwipe: {},
		mdEnhance: {
			include: {},
			gfm: true,
			attrs: true,
			footnote: true,
			mark: true,
			flowchart: true,
			align: true,
		},
	},
});
```

**config**

```
import { defineUserConfig } from 'vuepress';
import markdownItAttrs from 'markdown-it-attrs';
import abbr from 'markdown-it-abbr';

import { redirectPlugin } from 'vuepress-plugin-redirect';

import theme from './theme';

export default defineUserConfig({
	lang: 'ru-RU',
	title: 'Безопасность труда',
	description: 'материал инструктажа',
	theme,

	extendsMarkdown: (md) => {
		md.use(markdownItAttrs);
		md.use(abbr);
		md.use(require('markdown-it-ins'));
	},
	plugins: [redirectPlugin()],
});
```

**Comment by Mister-Hope** [comment]
What do you mean by "Text in the sidebar and in the title is doubled:"?

![image](https://user-images.githubusercontent.com/33315834/179196255-8f933d9f-3400-40a9-b73b-aee87ddb3d90.png)
![image](https://user-images.githubusercontent.com/33315834/179196313-f765f688-3b4d-436c-ba54-86cd3743e193.png)

If this is what you mean "doubled", you'd better check your own understanding

**Comment by Mister-Hope** [comment]
Oh I think I get you, a minial reproductin repo is required. Or a video about reproduction actions on official docs.

If you can only reproduce that in your own env, I am doubling this is your own problem.

**Comment by tadmi** [comment]
@Mister-Hope new site deployed on netlify: https://be-safety.netlify.app . After several page updates, the problem appears.

**Comment by Mister-Hope** [comment]
Provide me a reproduction by slightly modifying https://stackblitz.com/fork/vuepress-theme-hope

I have no access with your source code.

**Comment by Mister-Hope** [comment]
Provide me a reproduction by slightly modifying https://stackblitz.com/fork/vuepress-theme-hope

I have no access with your source code.

**Comment by Mister-Hope** [comment]
Also I see you importing mutiple plugins such as attrs and your own attrs plugins with markdown, you'd better disable them and only use the theme ones. I only accept issues with theme and plugins. And I won't waste time debuging any conflicts. If you met any, you should dig it yourself, and point me the issues straightly and possiable solutions.

**Comment by tadmi** [comment]
@Mister-Hope, i apologize for distracting attention to nonsense. The problem is observed only on netlify.

**Comment by Mister-Hope** [comment]
No need to apologize, you may check netilfy docs to see if it servers some assets specially.

For example, if you are serving file starting with _, you will get 404 with default settings with gh pages.

**Comment by EfChouTR** [comment]
@Mister-Hope it seems to be a bundler issue...locally it works fine but the transpiled assets run into duped text in the navbar for me

<img width="573" alt="image" src="https://user-images.githubusercontent.com/52469390/186572363-ac3ecb37-11e9-401d-bc0b-690c1e1edb41.png">

Our Vuepress site is cookie-cutter and has no special Vue components or config.

**Comment by EfChouTR** [comment]
Further investigation points to VueJS core SSR. See vuejs/core#6008 . They recently tried to fix it in 3.2.37, but the bug is still around.
